tests/model/model.cpp

```diff
@@ -465,7 +465,7 @@ TEST(Model, constructors)
 
     EXPECT_EQ(e, a);
 
-    //Testing copy constructor
+    // Testing copy constructor
```

Comment:
I guess it's not critical for our tests, but in our codebase we would have a full stop at the end of a comment.